integrations/test_simplesearcher_check_prf.py

```diff
@@ -0,0 +1,661 @@
+#
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import gzip
+import os
+import json
+import shutil
+import unittest
+from random import randint
+import tarfile
+from pyserini.util import download_url
+from integrations.utils import run_command, parse_score
+from integrations.simplesearcher_score_checker import SimpleSearcherScoreChecker
+
+
+class TestSearchIntegration(unittest.TestCase):
+    def setUp(self):
+        curdir = os.getcwd()
+        if curdir.endswith('integrations'):
+            self.pyserini_root = '..'
+            self.anserini_root = '../../anserini'
+        else:
+            self.pyserini_root = '.'
+            self.anserini_root = '../anserini'
+
+        self.tmp = f'{self.pyserini_root}/integrations/tmp{randint(0, 10000)}'
+
+        self.checker18 = SimpleSearcherScoreChecker(
+            index=os.path.join(self.anserini_root, 'indexes/lucene-index.core18.pos+docvectors+raw'),
+            topics=os.path.join(self.pyserini_root, 'tools/topics-and-qrels/topics.core18.txt'),
+            pyserini_topics='core18',
+            qrels=os.path.join(self.pyserini_root, 'tools/topics-and-qrels/qrels.core18.txt'),
+            eval=f'{self.pyserini_root}/tools/eval/trec_eval.9.0.4/trec_eval -m map -m P.30')
+
+        if os.path.exists(self.tmp):
+            shutil.rmtree(self.tmp)
+        else:
+            os.mkdir(self.tmp)
+
+        download_url('https://www.dropbox.com/sh/xo9v2glmwy3jrtn/AADdx7gdes6_pPAQKDPjgWDSa/core18.tar.gz?dl=1',
```

Comment:
Remove - assume index is there, throw exception if it isn't...

Reply:
Try and exception added. Python3 is also updated to python.